### ISSUE TITLE
[0.4.17 bug] Fatal error in void log_deprecated()

### ISSUE THREAD

**Issue by paramat** [minetest/minetest#6798]
```
2017-12-17 00:44:29: ERROR[Main]: In thread 7f8d58f33880:
2017-12-17 00:44:29: ERROR[Main]: /.../minetest/src/script/common/c_internal.cpp:9c:
void log_deprecated(lua_State*, const string&): A fatal error occured: lua_getstack() failed
```
When running an old mod that likely has deprecated code, when 'deprecated_lua_api_handling = log or error'.
Also reported by @sofar http://irc.minetest.net/minetest-dev/2017-12-16#i_5169873
sfan5 reports #6708 might fix it.

**Comment by rubenwardy** [comment]
#6708

**Comment by paramat** [comment]
See https://github.com/minetest/minetest/issues/7419#issuecomment-395186917
> Fixed in 0.5.0-dev a1598e1#diff-4e3f0a64f234ea624dbf1b329eb56bd9R157

If this is a duplicate of #7419 and is fixed in 0.5dev why is there a 0.5 milestone for this?
This issue is open for the bug in 0.4.17?

**Comment by rubenwardy** [comment]
This issue was made long before that commit

**Comment by SmallJoker** [comment]
Fixed in 0.4.17.1
